src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -38,21 +38,26 @@
 import api.support.builders.ItemBuilder;
 import api.support.builders.RequestBuilder;
 import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
 public class RequestsAPIRetrievalTests extends APITests {
   private static final String NEW_TAG = "new";
   private static final String IMPORTANT_TAG = "important";
   private static final String ONE_COPY_NUMBER = "1";
-  private static final String TWO_COPY_NUMBER = "2";
 
   @Test
   public void canGetARequestById() {
     UUID facultyGroupId = patronGroupsFixture.faculty().getId();
     UUID staffGroupId = patronGroupsFixture.staff().getId();
+    UUID isbnIdentifierId = UUID.randomUUID();
```

Comment:
Ideally identifier types would be managed in these tests similarly to how contributor and instance types are, in order to demonstrate how these values fit into the system and relate to other parts 

(and maybe to eventually allow these tests to be run against the real system again)